PurchasesCoreSwift/Purchasing/ReceiptRefreshPolicy.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ReceiptRefreshPolicy.swift
+//  PurchasesCoreSwift
+//
+//  Created by Juanpe Catalán on 7/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCReceiptRefreshPolicy) public enum ReceiptRefreshPolicy: Int {
```

Comment:
This enum is `public` because it must be visible outside. I think it could be a good idea to make it `internal` when the migration is done. WDYT?

Reply:
💯 You could add a //TODO: to the code, I have an issue #599 meant to address each todo like this 😄